docs/reference/snapshot-restore/index.asciidoc

```diff
@@ -0,0 +1,87 @@
+[[snapshot-restore]]
+= Snapshot and restore
+
+[partintro]
+--
+
+// tag::snapshot-intro[]
+A snapshot is a backup taken from a running Elasticsearch cluster. You can take
+a snapshot of individual indices or of the entire cluster and store it in a
+repository on a shared filesystem, and there are plugins that support remote
+repositories on S3, HDFS, Azure, Google Cloud Storage and more.
+
+Snapshots are taken incrementally. This means that when it creates a snapshot of
+an index, Elasticsearch avoids copying any data that is already stored in the
+repository as part of an earlier snapshot of the same index. Therefore it can be
```

Comment:
I'd move repository toward the end of this sentence.

Reply:
Reworked.